### PR TITLE
Make a mobile view for MonthView

### PR DESCRIPTION
**Pull request by kdjohnson**
I took the month view and removed the selected day preview. I gave the table full width. If an event occurs on a day there will be an icon in that cell. Which can be clicked then the user will taken to `ScheduleView`. Along, with clicking a user can hit space or enter to trigger the same behaviour. I'm having a tough time getting the selected day from `MobileMonthView` to be focused in the `ScheduleView`. This is something that I think needs to be there, but again it's proving to be more involved than I thought. Maybe it's something that can be more fleshed out later once the full fledged `MonthView` is more set in stone?  If you think it's necessary for being pulled into master I can keep hacking at it.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -17,9 +18,23 @@ class App extends Component {
     mobile: false
   }
 
+  updateMonthViewClicked = (year, month, week, day) => {
+    console.log("I've been called")
```
Comment (amgoodfellow) [inline src/App.js]: Maybe this `console.log()` can be removed? :)

File: src/components/Weekview.js [hunk 2]
```diff
@@ -153,8 +153,7 @@ class Weekview extends Component {
           classes
         )
       } catch (err) {
-        console.log(this.props.meetings)
-        //weekGrid = newWeekCol(null)
+        console.error(err)
```
Comment (amgoodfellow) [inline src/components/Weekview.js]: If this file remains unchanged, it will be a little easier to merge into master

**Review by amgoodfellow — COMMENTED** [review]
Just two small changes, other than that looks good. Thanks for adding Flow stuff to methods!

File: src/App.js (hunk 1 above)
Comment (kdjohnson) [reply src/App.js]: Changed.

File: src/components/Weekview.js (hunk 2 above)
Comment (kdjohnson) [reply src/components/Weekview.js]: Changed back to previous code.

**Review by amgoodfellow — APPROVED** [review]
Thanks! Looks wonderful :+1:

**Pull request merged**